### DISCUSSION TITLE
Action not run and no logs available (action/stale)

### DISCUSSION THREAD

**Discussion by bonomat** [community/community#26502] [Actions]
<p>Hi there,</p>
<p>we wanted to make use of <a href="https://github.com/actions/stale" rel="nofollow noopener">action/stale</a> in our repository in <a href="https://github.com/comit-network/comit-rs" rel="nofollow noopener">https://github.com/comit-network/comit-rs</a> however the action is never run. I don’t think it is a problem with the used action it self as the same action works just fine in a private repository (<a href="https://github.com/coblox/admin" rel="nofollow noopener">https://github.com/coblox/admin</a>) without a problem.</p>
<p>Can you help out please? </p>
<p>This is the source of the used action: <a href="https://github.com/comit-network/comit-rs/blob/master/.github/workflows/stale.yml" rel="nofollow noopener">https://github.com/comit-network/comit-rs/blob/master/.github/workflows/stale.yml</a> </p>
<p>Thank,</p>
<p>cheers,</p>
<p>Philipp</p>

**Comment by weide-zhou** [comment, marked as answer]
<p>Hi <a class="mention" href="/bonomat">@bonomat</a> ,</p>
<p>I forked your repo and the workflow is triggered successfully on my end. For debugging, I changed to use schedule 6 minutes per workflow, it works. Code as below:</p>
<pre><code>on:
  schedule:
- cron: '*/6 * * * *'
</code></pre>
<p>And then I changed back to your code and it works as well.</p>
<pre><code>on:
  schedule:
    - cron: 0 * * * *
</code></pre>
<p>I notice there is still no workflow in your repo, could you please change the trigger same with mine and check if it works?  </p>
<p>Regarding to the log, it depends on the action itself. If there is no log catch in the action, eg: lib/main.js (looks it’s hidden by Github), it won’t export logs. In my workflow, it only shows the execution step.</p>
<p><img src="https://user-images.githubusercontent.com/52772761/181098065-c9841dc1-2f61-49b7-a0af-61e7a465c21a.png" alt="stale.png" data-base62-sha1="r2OHNjqgFduzRYUyWIkt3vs3DVN" width="683" height="245"></p>

**Comment by bonomat** [comment]
<p>Hi <a class="mention" href="/weide-zhou">@weide-zhou</a> , </p>
<p>thanks for coming back to me that fast. </p>
<p>I’ve hanged the action according your proposal but it still does not seem to get fired:</p>
<p><a href="https://github.com/comit-network/comit-rs/blob/master/.github/workflows/stale.yml" rel="nofollow noopener">https://github.com/comit-network/comit-rs/blob/master/.github/workflows/stale.yml</a></p>
<p><a href="https://github.com/comit-network/comit-rs/actions?query=workflow%3Astale" rel="nofollow noopener">https://github.com/comit-network/comit-rs/actions?query=workflow%3Astale</a></p>
<p>Any idea of how to debug or move forward?</p>
<p>Cheers,</p>
<p>//edit// </p>
<p>I double checked and the actions permissions are enabled: </p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/3/371f55b074420385550471cb3fdf3889c29d02a0.png" data-download-href="/uploads/short-url/7RDovbLwvFfNVaQ53fioQne3s9q.png?dl=1" title="Screen Shot 2020-01-09 at 11.59.38.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/224613/181098069-922db2cf-e3cf-4ba5-8a45-36408188e011.png" alt="Screen Shot 2020-01-09 at 11.59.38.png" data-base62-sha1="7RDovbLwvFfNVaQ53fioQne3s9q" width="690" height="232" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/3/371f55b074420385550471cb3fdf3889c29d02a0_2_690x232.png, https://aws1.discourse-cdn.com/github/original/2X/3/371f55b074420385550471cb3fdf3889c29d02a0.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/3/371f55b074420385550471cb3fdf3889c29d02a0.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/3/371f55b074420385550471cb3fdf3889c29d02a0_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2020-01-09 at 11.59.38.png</span><span class="informations">780×263 25.7 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by weide-zhou** [comment]
<p>Hi <a class="mention" href="/bonomat">@bonomat</a>,</p>
<p>Could you please try another trigger, eg: push, is the workflow triggered succesfully? Then change back to ‘schedule’ for double check. I forked your repo, and it works for me. my repo here for your reference: <a href="https://github.com/weide-zhou/comit-rs/actions" rel="nofollow noopener">https://github.com/weide-zhou/comit-rs/actions</a></p>
<p>If it doesn’t work or even ‘push’ event doesn’t trigger , it’s recommended to ask private github support here: <a href="https://support.github.com/contact" rel="nofollow noopener">https://support.github.com/contact</a>.</p>
<p>Thanks.</p>

**Comment by bonomat** [comment]
<p>Hi <a class="mention" href="/weide-zhou">@weide-zhou</a> ,</p>
<p>thanks for your help so far. </p>
<p>I’ve changed it temporarily to being triggered by pushing to a particular branch. This worked.</p>
<p>Afterwards I’ve moved back to a trigger on schedule, which still does not help. </p>
<p>I’ve reached out to github support, let’s see if they can help. </p>
<p>I’ll keep you posted. </p>
<p>Cheers,</p>
<p>Philipp</p>

**Comment by bonomat** [comment]
<p><a class="mention" href="/weide-zhou">@weide-zhou</a> I aceepted your response as the solution. It’s working now and we can mark this as resolved :) </p>

**Comment by weide-zhou** [comment]
<p>Thanks <a class="mention" href="/bonomat">@bonomat</a> , glad to know it’s working for you now, happy coding!</p>
